master/src/main/java/org/evosuite/EvoSuite.java

```diff
@@ -215,9 +215,9 @@ public Object parseCommandLine(String[] args) {
 
             CommandLineParameters.addJavaDOptions(javaOpts, line);
 
-            if (TestSuiteWriterUtils.needToUseAgent() && Properties.JUNIT_CHECK) {
-                ClassPathHacker.initializeToolJar();
-            }
+//            if (TestSuiteWriterUtils.needToUseAgent() && Properties.JUNIT_CHECK) {
+//                ClassPathHacker.initializeToolJar();
```

Comment:
I understand that adding tools jar does not work with Java 9. But wouldn't removing it break EvoSuite on Java < 9?

Reply:
That was a redundant check. Line no. 160- 162 in the same file already takes care of it.